### DISCUSSION TITLE
WoL from Phone ?

### DISCUSSION THREAD

**Discussion by PohanThePagan** [seriousm4x/UpSnap#79] [Help wanted] [answered]
Hi i love the open source code that you wrote. I have old PC with Proxmox. I am planning to use your Docker on in. I needed the ability to turn on my main PC remotely preferably from my phone. 

I am new with proxmox and coding generay. So i wold love some advice !  I am running dinamic DNS that cold by usful to acces your docker from outside. But i man not sure how realy. Any advice or reference to other info serce wold by a big help! Than kyou

**Comment by seriousm4x** [comment, marked as answer]
I would suggest to use wireguard as it's easier than openvpn to set up. There are some great web ui projects like this one:
https://github.com/ngoduykhanh/wireguard-ui
with this docker-compose file:
https://github.com/ngoduykhanh/wireguard-ui/blob/master/examples/docker-compose/linuxserver.yml

Once thats up an running, port forward wireguard port 51820 to your wireguard host. Then you can just connect on your phone and access your local network.

**Reply by PohanThePagan** [reply]
ohh ok i will try it!
I will let you know how it ends up 
:D
